aztec/src/main/kotlin/org/wordpress/aztec/AztecText.kt

```diff
@@ -93,27 +106,45 @@ class AztecText : EditText, TextWatcher {
         removeTextChangedListener(this)
     }
 
+    fun setSelectedStyles(styles: ArrayList<TextFormat>) {
+        isNewStyleSelected = true
+        selectedStyles.clear()
+        selectedStyles.addAll(styles)
+    }
+
+    fun setOnSelectionChangedListener(onSelectionChangedListener: OnSelectionChangedListener) {
+        this.onSelectionChangedListener = onSelectionChangedListener
+    }
+
+    override fun onSelectionChanged(selStart: Int, selEnd: Int) {
+        super.onSelectionChanged(selStart, selEnd)
+        if (!consumeSelectionEvent) {
+            consumeSelectionEvent = false
```

Comment:
Oops. That's silly of me. Looks like I dropped the code that depends on it somewhere during the split of big PR. Fixed.